moduleroot/CONTRIBUTING.md

```diff
@@ -186,32 +195,78 @@ This will now download a pre-fabricated image configured in the [default node-se
 install puppet, copy this module and install its dependencies per [spec/spec_helper_acceptance.rb](./spec/spec_helper_acceptance.rb)
 and then run all the tests under [spec/acceptance](./spec/acceptance).
 
-Writing Tests
--------------
+## Writing Tests
 
-XXX getting started writing tests.
+### Unit Tests
 
-If you have commit access to the repository
-===========================================
+When writing unit tests for Puppet, [rspec-puppet][] is your best friend. It provides tons of helper methods for testing your manifests against a 
+catalogue (e.g. contain_file, contain_package, with_params, etc). It would be ridiculous to try and top rspec-puppet's [documentation][rspec-puppet_docs] 
```

Comment:
Yeah, but Puppet uses American style; consistency within our own docs is more important than consistency between our docs and someone else's. I was really just teasing---but on further reflection, we probably should change it.